jobs/job_test_utils.py

```diff
@@ -63,12 +85,36 @@ def assert_pcoll_equal(self, actual, expected):
     def assert_pcoll_empty(self, actual):
         """Asserts that the given PCollection is empty.
 
+        NOTE: This method doesn't raise the exception. It attaches a PTransform
+        to the "actual" PCollection which raises when the pipeline closes.
+
         Args:
             actual: PCollection. The PCollection generated by the test.
         """
         beam_testing_util.assert_that(
             actual, beam_testing_util.is_empty())
 
+    def new_model(self, model_type, **properties):
+        """Helper method for creating valid models with common default values.
+
+        Args:
+            model_type: type(*). Any subclass of BaseModel.
+            **properties: dict(str: *). Properties to assign to the model. By
+                default, this method will try to fill the required properties
+                with default values.
+
+        Returns:
+            *. A new instance of the given model type.
+        """
+        property_values = {
```

Comment:
What if there are fields that are required but there is no default for them?

Reply:
Then the method will still raise, and that's fine. I've added a raises section for this (even though it isn't raised explicitly).